Tests/HelperTest.php

```diff
@@ -241,20 +243,18 @@ public function should_ignore_irrelevant_attributes()
     /**
      * @test
      */
-    public function should_process_jobs()
+    public function should_process_job_parameters()
     {
-        $command = $this->createMock(Command::class);
-        $command->expects($this->once())->method('getName')->willReturn('my:job');
+        $command = new ProcessTestCommand();
```

Comment:
Please continue using the mock, it is vastly preferrable to resource files. We are not trying to test the annotation reader here, that doctrine's part. (Note: for attributes I had to create resource files, as I haven't found a way to mock reflection->getAttributes)